src/components/settings/generalsettings/ResolutionSettings.js

```diff
@@ -1,324 +1,262 @@
-        DiscoverResolution: {
-          x: 81,
-          y: 209,
-          label: 'Resolution Settings',
-          type: ResolutionTile
-        },
-        ResolutionControl: {
-         x: 85,
-         y: 325,
-         text: {
-           text: 'Available Resolutions',
-           fontSize: 34,
-           textColor: Colors.TRANSPARENT_GREY,
-           fontFace: 'Regular'
+         },
+         DiscoveredResolution: {
+           type: Lightning.components.ListComponent,
+           x: 81,
+           y: 398,
+           w: 800,
+           h: 1080,
+           visible: false,
+           itemSize: 120,
+           horizontal: false,
+           roll: true,
+           rollMax: 1080,
+           rollMin: 0,
+           invertDirection: true
          }
-        },
-        /*DiscoveredResolution: {
-          type: ScrollableListResolution,
-          x: 81,
-          y: 398,
-          w: 800,
-          h: 1080,
-          visible: false,
-          itemSize: 120,
-          horizontal: false,
-          spacing: 20,
-          roll: true,
-          rollMax: 800,
-          rollMin: 0,
-          invertDirection: true
-        }*/
-        DiscoveredResolution: {
-          type: Lightning.components.ListComponent,
-          x: 81,
-          y: 398,
-          w: 800,
-          h: 1080,
-          visible: false,
-          itemSize: 120,
-          horizontal: false,
-          roll: true,
-          rollMax: 1080,
-          rollMin: 0,
-          invertDirection: true
-        }
+       },
+       ThunderResolutionService: {
+      type: ThunderResolutionService
       }
+     }
+   }
+ 
+   _init() {
+     this.availableresolution = "";
+     /*this.setbestresolution();*/
+     
+     if(Storage.get("lastsetresolution"))
+     {
+     console.log("ResTag:lastsetresolution from storage :",Storage.get("lastsetresolution"))
+     this.currentresolution =Storage.get("lastsetresolution")
+     }
+     else
+     {
+     this.tag('ThunderResolutionService').getcurrentresolution().then(data => {
+     this.currentresolution = data
+     });
+     }
+     this.scanFlag = 1;
+     this.removeChildFlag = 0;
     }
-  }
-
-   _construct() {
-     Log.info('Resolution Component ')
-       this.config = {
-         host: '127.0.0.1'
+ 
+   _active() {
+      this.flag = 0;
+     this._setState('DiscoverResolution')
+   }
+  
+ 
+   startScanResolution() {
+     Log.info('ResolutionUI: startScanResolution enter.')
+     this.tag('ResolutionControl').patch({
+         text: { text: 'Searching for available resolutions...' }
+     })
+    
+     this.tag('ThunderResolutionService').getavailableresolutions().then(data => {
+     console.log("Rtag:available resolution", data);
+     this.availableresolution = data;
+     
+         setTimeout( ()=> {
+       Log.info('ResoltionUI: Available Resolutions:', this.availableresolution);
+       if (this.availableresolution === "") {

```

Comment:
It is a standard to use single quotes  instead of double quotes for strings in JS. Discuss with Krishnapriya whether  it is required